jenkins-plugin/src/test/java/org/jenkinsci/plugins/pipeline/maven/DependencyGraphTest.java

```diff
@@ -106,9 +104,6 @@ public void verify_downstream_simple_pipeline_trigger() throws Exception {
         jenkinsRule.waitUntilNoActivity();
 
         WorkflowRun mavenWarPipelineLastRun = mavenWarPipeline.getLastBuild();
-        SqlTestsUtils.dump("select * from job_generated_artifacts", ((PipelineMavenPluginJdbcDao)GlobalPipelineMavenConfig.get().getDao()).getDataSource(), System.out);
```

Comment:
I removed these lines, as I have moved the `getDataSource()` into the abstract DAO implementation. If they are really important, I can implement it in another way, but since it is just debugging info, its probably not that critical.